examples/forward/plot_transform.py

```diff
@@ -0,0 +1,65 @@
+import os.path as op
+import numpy as np
+
+import mne
+from mne.forward import make_field_map, transform_instances
+
+
+path = mne.datasets.sample.data_path()
+evoked_fname = op.join(path, 'MEG', 'sample', 'sample_audvis-ave.fif')
+trans_fname = op.join(path, 'MEG', 'sample', 'sample_audvis_raw-trans.fif')
+subjects_dir = op.join(path, 'subjects')
+
+condition = 'Left Auditory'
+evoked = mne.read_evokeds(evoked_fname, condition=condition, baseline=(-0.2, 0.0))
+
+# Compute the field maps to project MEG and EEG data to MEG helmet
+# and scalp surface
+maps = make_field_map(evoked, trans_fname, subject='sample',
+                      subjects_dir=subjects_dir,  meg_surf='head')
+p_field = evoked.plot_field(maps, time=0.11)
+
+
+p_helmet = mne.viz.plot_trans(evoked.info, trans_fname, 'sample', subjects_dir,
+                              ch_type='meg', coord_frame='meg')
+
+
+# let's assume you have multiple runs and your subject moved :(
+evoked_rot = evoked.copy()
+trans_rot = evoked_rot.info['dev_head_t']['trans']
+
+rot = mne.transforms.rotation(x=0, y=np.pi/16, z=0)
```

Comment:
I would use translation because it's more intuitive to interpret. Something like: `translate = mne.transforms.translation(x=0, y=0, z=0.05)` looks nice for me. Let's get rid of the `plot_trans` calls. The field map plot shows it anyway, right?